hal/hal_uv_client_usb.c

```diff
@@ -196,18 +196,29 @@ static int usb_hotplug_check(adb_client_usb_t* client) {
 
 static int usb_uv_write(adb_client_t *c, apacket *p) {
     int ret;
-    uv_buf_t buf;
+    uv_buf_t buf[2];
```

Comment:
@JianyuWang0623 let's drop the patch and reverted patch

Reply:
> @JianyuWang0623 let's drop the patch and reverted patch

get, will update later